msyncd/AccountsHelper.cpp

```diff
@@ -186,7 +189,7 @@ void AccountsHelper::slotAccountEnabledChanged(const QString &serviceName, bool
             foreach(SyncProfile *profile, profiles)
             {
                 // See if the service name matches
-                if(serviceName == profile->name())
+                if(serviceName == profile->key(REMOTE_SERVICE_NAME))
                 {
```

Comment:
Why this change ? if the profile exit already created when account is loaded above, there's no guarantee that will be the REMOTE_SERVICE_NAME as far as I can see.